lib/xml_security.rb

```diff
@@ -255,12 +255,12 @@ def validate_document_with_cert(idp_cert)
         begin
           cert = OpenSSL::X509::Certificate.new(cert_text)
         rescue OpenSSL::X509::CertificateError => _e
-          return append_error("Certificate Error", soft)
+          return append_error("Document certificate error", soft)
```

Comment:
Please keep the "Certificate Error" message, maybe there are integrations already listening for such string.